### PR TITLE
optimize S3 partitioning

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

We inherited a way of naming s3 objects which might have been fine in its original context, but is not optimal for AWS.  For S3, we should not be prefixing all of our s3 objects with "service-uuid" because it pushes everything onto one partition, which will affect search speeds as we scale.  Change the name format to "uuid-service"

Note:  This PR is has a matching PR on the admin side.  They should be checked in at the same time.  

## Security Considerations

N/A

### CODE REVIEW

File: app/aws/s3.py [hunk 1]
```diff
@@ -257,7 +278,7 @@ def get_job_from_s3(service_id, job_id):
                 "SlowDown",
             ]:
                 current_app.logger.exception(
-                    f"Retrying job fetch {FILE_LOCATION_STRUCTURE.format(service_id, job_id)} retry_count={retries}",
+                    f"Retrying job fetch  retry_count={retries}",
```
Comment (ccostino) [inline app/aws/s3.py:281]: Are we missing a variable to include here so we know what we're retrying?  I'm not sure what it'd be, I wonder if it's something to key off of with the `obj` variable, such as a property or something?

```suggestion
                    f"Retrying job fetch {VARIABLE HERE} retry_count={retries}",
```

Comment (ccostino) [reply app/aws/s3.py:281]: Or maybe it's what you have below, `{service_id}`?

**Review by ccostino — COMMENTED** [review]
Thanks, @terrazoon!

Just one quick question about one of the logging statements here - it might be missing some info.

Noted that we'll have to update again after about a week or two!

File: app/aws/s3.py (hunk 1 above)
Comment (terrazoon) [reply app/aws/s3.py]: fixed

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Review by A-Shumway42 — APPROVED** [review]
No concerns here. LGTM.

**Pull request merged**